isyfact-standards-doc/src/docs/antora/modules/referenzarchitektur/pages/software-technisch/backend/serviceschicht.adoc

```diff
@@ -0,0 +1,119 @@
+= Serviceschicht
+
+include::documentation-guide::partial$licence.adoc[]
+
+// tag::serviceschicht-definition[]
+
+Die Serviceschicht stellt Services von Backends innerhalb einer Systemlandschaft zur Verfügung.
+Sie regelt die Autorisierung von Aufrufen, validiert die übergebenen Daten und ruft die entsprechenden fachlichen Operationen des Anwendungskerns auf.
+Der Aufbau der Serviceschicht sowie ihre Kommunikation mit anderen Schichten ist in <<systemarchitektur-serviceschicht,folgender Grafik>> dargestellt.
+
+[[systemarchitektur-serviceschicht]]
+.Einbettung der Serviceschicht in die Systemarchitektur
+image::software-technisch/backend/service.dn.svg[width=80%,align=center]
```

Comment:
Es fällt auf, dass in dieser Grafik "service.dn.svg" die Service-Consumer als Teil des Anwendungskerns (blauer Kasten) gesehen werden.

in "" jedoch gehören sie zum SGW

Reply:
Es handelt sich auch nicht um dieselbe Komponente, sondern um Komponenten mit derselben Funktion. Siehe auch: https://isyfact.github.io/isyfact-standards-doku/dev/referenzarchitektur/software-technisch/backend/anwendungskern.html#service-consumer

Reply:
Das ist schon klar, (Service-Consumer-Komponente im AWK ruft Service-Consumer-Komponente im SGW)
aber es ist trotzdem verwirrend: denn es wird nirgendwo darauf hingewiesen dass es an den betreffenden Stellen um unterschiedliche Service-Consumer geht.